### ISSUE TITLE
Deprecation error on run Symfony\Component\Process\Process

### ISSUE THREAD

**Issue by BobRay** [Codeception/Codeception#5353]
#### What are you trying to achieve?
Run Codeception Acceptance test
#### What do you get instead?
The test runs, but issues this warning:

2x DEPRECATION: Passing a command as string when creating a "Symfony\Component\Process\Process" instance is deprecated since Symfony 4.2, pass it as an array of its arguments instead, or use the "Process::fromShellCommandline()" constructor if you need features provided by the shell. C:\xampp\htdocs\addons\vendor\symfony\process\Process.php:147

> Provide console output if related. Use `-vvv` mode for more details.
See above

### Details

The warning does not appear when running in PhpStorm, only when running from the command line.

* Codeception version:  2.5.2
* PHP Version: 7.2
* Operating System: Windows 10
* Installation type: Composer
* List of installed packages (`composer show`)

behat/gherkin                      v4.5.1  Gherkin DSL parser for PHP 5.3
codeception/codeception            2.5.2   BDD-style testing framework
codeception/phpunit-wrapper        7.6.1   PHPUnit classes used by Codeception
codeception/stub                   2.0.4   Flexible Stub wrapper for PHPUnit's Mock Builder
doctrine/instantiator              1.1.0   A small, lightweight utility to instantiate object...
facebook/webdriver                 1.6.0   A PHP client for Selenium WebDriver
guzzlehttp/guzzle                  6.3.3   Guzzle is a PHP HTTP client library
guzzlehttp/promises                v1.3.1  Guzzle promises library
guzzlehttp/psr7                    1.5.2   PSR-7 message implementation that also provides co...
myclabs/deep-copy                  1.8.1   Create deep copies (clones) of your objects
phar-io/manifest                   1.0.3   Component for reading phar.io manifest information...
phar-io/version                    2.0.1   Library for handling version information and const...
phpdocumentor/reflection-common    1.0.1   Common reflection classes used by phpdocumentor to...
phpdocumentor/reflection-docblock  4.3.0   With this component, a library can provide support...
phpdocumentor/type-resolver        0.4.0
phpspec/prophecy                   1.8.0   Highly opinionated mocking framework for PHP 5.3+
phpunit/php-code-coverage          6.1.4   Library that provides collection, processing, and ...
phpunit/php-file-iterator          2.0.2   FilterIterator implementation that filters files b...
phpunit/php-text-template          1.2.1   Simple template engine.
phpunit/php-timer                  2.0.0   Utility class for timing
phpunit/php-token-stream           3.0.1   Wrapper around PHP's tokenizer extension.
phpunit/phpunit                    7.5.1   The PHP Unit Testing framework.
psr/http-message                   1.0.1   Common interface for HTTP messages
ralouphie/getallheaders            2.0.5   A polyfill for getallheaders.
sebastian/code-unit-reverse-lookup 1.0.1   Looks up which function or method a line of code b...
sebastian/comparator               3.0.2   Provides the functionality to compare PHP values f...
sebastian/diff                     3.0.1   Diff implementation
sebastian/environment              4.0.1   Provides functionality to handle HHVM/PHP environm...
sebastian/exporter                 3.1.0   Provides the functionality to export PHP variables...
sebastian/global-state             2.0.0   Snapshotting of global state
sebastian/object-enumerator        3.0.3   Traverses array structures and object graphs to en...
sebastian/object-reflector         1.1.1   Allows reflection of object attributes, including ...
sebastian/recursion-context        3.0.0   Provides functionality to recursively process PHP ...
sebastian/resource-operations      2.0.1   Provides a list of PHP built-in functions that ope...
sebastian/version                  2.0.1   Library that helps with managing the version numbe...
symfony/browser-kit                v4.2.2  Symfony BrowserKit Component
symfony/console                    v4.2.2  Symfony Console Component
symfony/contracts                  v1.0.2  A set of abstractions extracted out of the Symfony...
symfony/css-selector               v4.2.2  Symfony CssSelector Component
symfony/dom-crawler                v4.2.2  Symfony DomCrawler Component
symfony/event-dispatcher           v4.2.2  Symfony EventDispatcher Component
symfony/finder                     v4.2.2  Symfony Finder Component
symfony/polyfill-ctype             v1.10.0 Symfony polyfill for ctype functions
symfony/polyfill-mbstring          v1.10.0 Symfony polyfill for the Mbstring extension
symfony/process                    v4.2.2  Symfony Process Component
symfony/yaml                       v4.2.2  Symfony Yaml Component
theseer/tokenizer                  1.1.0   A small library for converting tokenized PHP sourc...
webmozart/assert                   1.4.0   Assertions to validate method input/output with ni...

* Suite configuration:

```yml
# Codeception Test Suite Configuration
#
# Suite for acceptance tests.
# Perform tests in browser using the WebDriver or PhpBrowser.
# If you need both WebDriver and PHPBrowser tests - create a separate suite.

actor: AcceptanceTester
modules:
    enabled:
        - \Helper\Acceptance
        - WebDriver
        - Asserts
    config:
        WebDriver:
            url: 'http://localhost/test/'
            browser: 'chrome'
            window_size: 'maximize'
            wait: 5
            # window_size: false
            #capabilities:
               # chromeOptions:
                    #args: ["--headless", "--disable-gpu", "--disable-extensions"]

extensions:
    enabled:
        - Codeception\Extension\RunProcess:
              - java -jar c:/Users/BobRay/Downloads/Selenium/selenium-server-standalone-3.9.0.jar
              - C:/Users/BobRay/Downloads/selenium-drivers/chromedriver.exe
              # -  C:/Users/BobRay/Downloads/selenium-drivers/geckodriver.exe
```

**Comment by Naktibalda** [comment]
`symfony/process` library is used by [RunProcess](https://github.com/Codeception/Codeception/blob/2.5.2/ext/RunProcess.php#L80) and [RunBefore](https://github.com/Codeception/Codeception/blob/2.5.2/ext/RunBefore.php#L83) extensions.

`symfony/process` 2.7 and 3.* versions only accept string, so in order to support Symfony 5.0 which will be released at the end of this year, we will have to either detect version of library or drop support for symfony 2 and 3.

**Comment by denys-filchakov** [comment]
Hello, @Naktibalda 

Symfony already using the library process where was replaced the first argument in constructor of class Process was replaced in favor of array.

Right now import of dump not working with newest versions codeception 3.0 and sympfony:process ^4.2

**Comment by Naktibalda** [comment]
Support for symfony/process 5.0 was implemented in Codeception 4.0.0
